src/frontend/Sider/Behandling/Stønadsvilkår/DagligReise/EndreVilkår/EndreVilkårsvurderinger/EndreVurderinger.tsx

```diff
@@ -104,7 +104,7 @@ export const EndreVurderinger: React.FC<Props> = ({
         <VStack gap="4">
             {aktiveVurderinger.get(RegelIdDagligReise.AVSTAND_OVER_SEKS_KM) && (
                 <EndreDelvilkår
-                    label="Er reiseavstanden over 6km?"
+                    label="Er reiseavstanden minst 6km?"
```

Comment:
```suggestion
                    label="Er reiseavstanden minst 6 km?"
```
eller skal vi ha 
```suggestion
                    label="Er reiseavstanden 6 km eller mer?"
```
også her kanskje?